README.md

```diff
@@ -1 +1,172 @@
-# Microverse-first-capstone-project
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+  <h3><b>First Capstone Project</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+# 📖 [PIANO CONCERT PROJECT] <a name="about-project"></a>
+
+My website design inspired by Behance.I makes more styling for better view and add javascript to fast.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML</a></li>
+  </ul>
+   <ul>
+    <li><a href="#">CSS</a></li>
+  </ul>
+   <ul>
+    <li><a href="#">Javascript</a></li>
+  </ul>
+</details>
+          
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- **RWD for any devices**
+- **Use Flexbox to place elements in the page**
+- **Use GitHub Pages to deploy web pages**
+- **Use JavaScript to manipulate DOM elements**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
+
```

Comment:
- [ ] Please use GitHub Pages to deploy this project and update the this section with the correct deployment link. It is a project  requirement to deploy the project 👍🏾